### ISSUE TITLE
EM section to do list

### ISSUE THREAD

**Issue by Pbellive** [ubcgif/gpg#134]
Basic principles:
- [ ] Flesh out intro paragraph
- [ ] Update figures
- [ ] Rip Lens' law section from em.geosci
- [ ] Take some em.geosci material for three-loop example (can't just link to it, em.geosci too detailed)

Survey:

- [ ] Add images
- [ ] Move primary field in the earth section to basic principles?

Data and processing: It's not clear to me how to separate content between these two sections.

Interpretation:

- [ ] Add images
- [ ] Link to apparent conductivity app
- [ ] Expand?
- [ ] Expo site case history?

In all cases lecture notes can be guide for level at which material should be presented. @fourndo, @lheagy , @dougoldenburg feel free to add to the to-do list/check things off.

**Comment by fourndo** [comment]
That's great, thanks for starting this @Pbellive .
You saw the branch #133 yes?
I have to meet with Doug now, but can come down to go over this after

**Comment by Pbellive** [comment]
Yep, I've got the new branch. Thanks for setting that up @fourndo. I'll be in my office if you want to come down after talking to Doug.
